src/pages/About/index.js

```diff
@@ -30,50 +50,60 @@ const About = () => {
                     About Loofah
                 </Typography>
                 <Typography variant="body1">
-                    Loofah is a tool designed to help you understand the capabilities
-                    expected of you as an engineer at Deloitte Digital, as you progress through the organisation.
+                    Loofah is a tool designed to help you understand the capabilities expected of you as an engineer at
+                    Deloitte Digital, as you progress through the organisation.
                 </Typography>
 
-                <Typography variant="h5" className={classes.subHeading}>
-                    Understanding the capabilities
+                <List>
+                    <ListItem className={[classes.listItem, "MuiTypography-body1"]}>
+                            Read the <Link to={{pathname: "/pathway"}} className={classes.link}>Engineering Career Pathway</Link> to
+                            get a sense of the high-level expectations at each grade.
+                   </ListItem>
+                    <ListItem className={[classes.listItem, "MuiTypography-body1"]}>
+                            Dive into the <Link to={{pathname: "/skills", state: {grade: "analystDeveloper"} }} className={classes.link}>Engineering Skills</Link> to
```

Comment:
This line should be showing a linting error, is it? It's super long

Reply:
Was having trouble to get spaces around the link when i put the link on a new line in the html...

Have reduced the line length although there might be a better way to do this.